doc/en/how-to/capture-stdout-stderr.rst

```diff
@@ -4,6 +4,15 @@
 How to capture stdout/stderr output
 =========================================================
 
+Pytest can intercept stdout and stderr. Using the `--capture=` command-line
+argument (described below) allows pytest to add captured output to reports
+generated after tests. The reports can customized by `-r` (described below).
+
+Built-in fixtures (also described below) can be passed to tests as arguments
+to capture stdout and stderr and make it accessible to the developer during
+the test for inspection. It will not be available for reports if it is captured,
+unless it is re-printed with capturing turned off (see below).
+
```

Comment:
final draft in last commit, rewritten after betting understanding the behavior due to the changes